DatWeatherDoe/AppDelegate.swift

```diff
@@ -231,6 +219,40 @@ class AppDelegate: NSObject, NSApplicationDelegate, CLLocationManagerDelegate {
         }
     }
     
+    // MARK: Menu
+    
+    private func createMenu() -> NSMenu {
+        let menu = NSMenu()
+        menu.addItem(
+            withTitle: NSLocalizedString("Unknown Location", comment: "Unknown weather location"),
+            action: nil,
+            keyEquivalent: ""
+        )
+        menu.addItem(.separator())
+        menu.addItem(
+            withTitle: NSLocalizedString("See Full Weather", comment: "See Full Weather"),
```

Comment:
hey @Marc66 , I have included a French translated for this new string. Can you please validate? 🙏